js/weird-string-case.js

```diff
@@ -0,0 +1,15 @@
+// https://www.codewars.com/kata/52b757663a95b11b3d00062d/train/javascript
+
+function toWeirdCase(string){
+  const newArr = string.split('').map((letter, i) => {
```

Comment:
How about returning the result immediately? It's not needed to have this variable, you could just return the chain of operations